src/imars3d/backend/autoredux.py

```diff
@@ -23,7 +24,7 @@ def auto_reduction_ready(data_file: str) -> bool:
     raise NotImplementedError
 
 
-def load_template_config() -> dict:
+def load_template_config(config_path) -> dict:
```

Comment:
can you give a type hint for input argument `config_path`? It should accept both `str`and `pathlib.Path`